development/development/phpstorm.rst

```diff
@@ -70,14 +70,10 @@ The default inspection settings should work just fine. However there are a coupl
 
 * If PhpStorm is not connected to your database server, you may want to turn off **SQL > SQL Dialect Detection** and **SQL > No data sources configured** warnings.
 * phpBB uses fully qualified namespaces, so you can turn off this inspection warning **PHP > Code Style > Unnecessary fully qualified name**.
-* You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable JSCS, JSHint, JSlint and/or ESLint. phpBB comes with configuration files for JSHint and JSCS. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. To do so, point PhpStorm to the custom configuration files under **Language & Frameworks > JavaScript > Code Quality Tools**. Point JSHint to `.jshintrc` and point JSCS to `.jscsrc`.
+* You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable ESLint. phpBB comes with configuration for XO.js which is a wrapper for ESLint. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. XO.js integration does not exsits for PhpStorm. XO.js is a wrapper tool for ESlint that is run through NodeJS on the command line. To use XO.js, make sure the NodeJS framework is downloaded and installed on your computer. Then you can install XO.js via the npm package manager and configure the ESlint options in PhpStorm. You can follow these  `Instructions <https://github.com/jamestalmage/xo-with-webstorm>`_ to get everything setup.
 
 .. warning:: Stylelint is not bundled with PhpStorm. The Stylelint tool is run through NodeJS. To use Stylelint, make sure the NodeJS framework is downloaded and installed on your computer. Then you can follow these `Instructions <https://www.jetbrains.com/help/phpstorm/stylelint.html>`_.
 
-.. warning:: JSCS is not bundled with PhpStorm. The JSCS tool is run through NodeJS. To use JSCS, make sure the NodeJS framework is downloaded and installed on your computer. Then you can install JSCS via the npm package manager and configure the JSCS options in PhpStorm to use the JSCS package. For more information review `PhpStorm's documentation <https://www.jetbrains.com/help/phpstorm/10.0/using-javascript-code-quality-tools.html#JSCS>`_.
-
-.. note:: phpBB uses jQuery. The Javascript inspections need to be made aware of jQuery to avoid any false warnings/errors. To do this, simply go to **Languages & Frameworks > JavaScript > Libraries** and enable jQuery. If jQuery is not in the list, you can use the "Download" button to download a copy of jQuery to PhpStorm.
```

Comment:
Why are you removing this? I think this is still relevant.

Reply:
No they are not they are discontinued and replaced by raking which is why we are switching

Reply:
I think VSE is referring to the jQuery part specifically, not JSCS.

Reply:
Yep jQuery is still very much a part and the instructions on how to make phpstorm recognize mustn't be deleted here.

Reply:
i deleted it because its handled by eslint/xojs via the package.json already. I am not familiar with phpstorm as I do not use it so i will let that up to those who do to know better if its still nessecary once we get xojs working with phpstorm

Reply:
That comment about jQuery has nothing to do with XO or ESLint or anything. XO and ESLint are linters. That setting is so PHPStorm will recognize jQuery code, and not mark it as erroneously written javascript, which is why it must stay. Do not delete it as it has nothing to do with linting.